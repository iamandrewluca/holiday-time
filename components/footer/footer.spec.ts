import { describe, it, expect } from '@jest/globals'
import { render } from '@testing-library/vue'
import Footer from './footer.section.vue'

describe('Footer', () => {
  it('should render and match snapshot', () => {
    const { baseElement } = render(Footer)
    expect(baseElement).toMatchSnapshot()
  })
})
