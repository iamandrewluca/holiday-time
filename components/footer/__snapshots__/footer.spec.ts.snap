// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render and match snapshot 1`] = `
<body>
  <div>
    <footer
      class="footer"
    >
      <div>
        <span>
          ✨✨✨
        </span>
         
        <span
          class="footer__text"
        >
          Have fun in 2021
        </span>
         
        <span>
          ✨✨✨
        </span>
      </div>
    </footer>
  </div>
</body>
`;
